### ISSUE TITLE
License mentioned in setup.py is still listed as GPLv2

### ISSUE THREAD

**Issue by schnorea** [secnot/rectpack#20]
Inside the setup.py the library listed there is GPLv2.  A bit of a nit but our OSS scanner picked it up.

**Comment by secnot** [comment]
Hi 

I forgot to change it when it was released under Apache-2. I will update the github repository today but It will be a little longer until the new version is uploaded to pypi.

Thank you.

**Comment by schnorea** [comment]
Thanks for the quick turn around!

**Comment by Sammbale7** [comment]
Hi. My friends and I have an AI group coursework, we are discussing the bin packing problem. We wanted to find out if we are allowed to make modifications to your algorithms and use it for our course project demonstration - we will reference it of course.

**Comment by secnot** [comment]
Yes you can modify the package, it's released under Apache License 2.0 so you only need to include the license, and if I remember correctly add a notice to the files that you modified.

**Comment by Sammbale7** [comment]
Alright, thank you.
